service/actions/users.js

```diff
@@ -149,14 +149,30 @@ export function logout() {
 }
 
 export function getProfiles(offset, limit = Constants.PROFILE_CHUNK_SIZE) {
-    return bindClientFunc(
-        Client.getProfiles,
-        UsersTypes.PROFILES_REQUEST,
-        [UsersTypes.RECEIVED_PROFILES, UsersTypes.PROFILES_SUCCESS],
-        UsersTypes.PROFILES_FAILURE,
-        offset,
-        limit
-    );
+    return async (dispatch, getState) => {
```

Comment:
I needed to return to profiles to check if the object is empty to stop future unneeded requests to the server